homeassistant/components/flipr/sensor.py

```diff
@@ -0,0 +1,86 @@
+"""Sensor platform for the Flipr's pool_sensor."""
```

Comment:
This PR contains 2 platforms: binary_sensor & sensor.
Please limit the initial PR to add an integration to a single platform.

Reply:
It would require some rework to remove the binary_sensor (which is less important) that I would have to put in another PR right after this one and It would imply to retest everything.
Is it a blocker one or could you accept it as is ?

Reply:
Yes, this is a blocker and is described in our developer documentation:

<https://developers.home-assistant.io/docs/creating_component_code_review#5-make-your-pull-request-as-small-as-possible>

Reply:
Done